.github/workflows/check_orgs.yml

```
@@ -0,0 +1,28 @@
+name: Check publicly listed organizations
+on:
+  repository_dispatch:
+  push:
+    paths:
+      - community/organizations.md
+jobs:
+  check_orgs:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+        with:
+          persist-credentials: false
+          fetch-depth: 1
+
+      - uses: julia-actions/setup-julia@v1
+        with:
+          version: 1.9
+
+      - uses: julia-actions/julia-buildpkg@v1
+        with:
+          project: tools/check_orgs
+
+      - run: julia check_orgs.jl
```

I guess you want to specify the project, no?
```suggestion
      - run: julia --project=. --color=yes check_orgs.jl
```